appveyor.yml

```diff
@@ -3,14 +3,15 @@ image: Visual Studio 2017
 environment:
   major: 3
   minor: 0
-  release: 0
+  release: 1
```

Comment:
Even though the code is not changing, the package will be different .So I'd like to see it somewhere in the release version